c7n/resources/awslambda.py

```diff
@@ -120,6 +120,35 @@ def get_iam_arns(self, resources):
         return [r['Role'] for r in resources]
 
 
+@AWSLambda.filter_registry.register('url-config')
+class URLConfig(ValueFilter):
+
+    annotation_key = "c7n:UrlConfig"
+    schema = type_schema('url-config', rinherit=ValueFilter.schema)
+    schema_alias = False
+    permissions = ('lambda:GetFunctionUrlConfig',)
+
+    def process(self, resources, event=None):
+        client = local_session(self.manager.session_factory).client('lambda')
+
+        def _augment(r):
+            try:
+                r[self.annotation_key] = self.manager.retry(
+                    client.get_function_url_config, FunctionName=r['FunctionArn'])
+                r[self.annotation_key].pop('ResponseMetadata')
+            except client.exceptions.ResourceNotFoundException:
+                r[self.annotation_key] = {}
+            return r
+
+        with self.executor_factory(max_workers=3) as w:
```

Comment:
this should ideally be at count 2 i think

Reply:
Addressed, thanks